.github/compatibility-test-matrices/release-v6.1.x/client-chain-a.json

```diff
@@ -3,21 +3,13 @@
     "release-v6.1.x"
   ],
   "chain-b": [
-    "v7.3.0",
-    "v7.2.1",
-    "v7.2.0",
-    "v6.2.0",
-    "v6.1.1",
-    "v5.3.1",
-    "v5.2.1",
-    "v4.4.2",
     "release-v6.1.x"
   ],
   "entrypoint": [
     "TestClientTestSuite"
   ],
   "test": [
-    "TestRecoverClient_Succeeds",
```

Comment:
I remove `TestRecoverClient_Succeeds` from all releases < than v8.0.0, since the gov v1 message will not be available in those releases.